dipy/viz/interactor.py

```diff
@@ -36,6 +36,13 @@ def abort(self):
         """ Aborts the event i.e. do not propagate it any further. """
         self._abort_flag = True
 
+    def done(self):
```

Comment:
I suggest renaming this to something like `reset`. `Event`'s other methods are verbs.